src/index.ts

```diff
@@ -1,5 +1,6 @@
 import { readFileSync } from "fs";
 import * as core from "@actions/core";
+import * as github from "@actions/github";
```

Comment:
The import statement for `@actions/github` should be placed in a more logical order, typically after the core imports and before any third-party libraries.